e2e/dpos-2-validators.toml

```diff
@@ -28,11 +43,24 @@
 
 [[TestCases]]
   RunCmd = "{{ $.LoomPath }} dpos3 delegate {{index $.NodeAddressList 0}} 11 -k {{index $.NodePrivKeyPathList 0}}"
+  
 
 [[TestCases]]
   RunCmd = "check_validators"
   Condition = "contains"
   Expected = ["{{index $.NodePubKeyList 0}}"]
+  ExpectedJson = [
+  "jsonrpc", "2.0",
+  "result.validators.#.pub_key.value","{{index $.NodePubKeyList 0}}"
+]
+
+[[TestCases]]
+  RunCmd = "check_validators"
+  Condition = "arithmetic"
+  KeysValues = [
+    "result.validators.#.voting_power",">","1250000000000",
+    "result.validators.#(pub_key.value=={{index $.NodePubKeyList 0}}).voting_power","<=","1260000000000"
+]
```

Comment:
Can you combine `KeyValues` & `ExpectedJSON` into one thing, so it would look like this?
```
ExpectedJSON = [
    "statistics.#.address.local", "==", "{{index $.NodeBase64AddressList 1}}"
    "result.validators.#.voting_power",">","1250000000000"
]
```
Basically run the template evaluation on the right-hand operand, then evaluate the whole expression using `gval.Evaluate`.